### PR TITLE
Upgraded to v4.1.0 and fixed AddClaim

### PR DESCRIPTION
**Pull request by TimurZanagar**

### CODE REVIEW

File: guides/getting-started.md [hunk 1]
```diff
@@ -149,13 +149,13 @@ If you don't want to start from one of the recommended samples, you'll need to:
             var identity = new ClaimsIdentity(TokenValidationParameters.DefaultAuthenticationType, Claims.Name, Claims.Role);
 
             // Use the client_id as the subject identifier.
-            identity.AddClaim(Claims.Subject,
+            identity.AddClaim(new Claim(Claims.Subject,
                 await _applicationManager.GetClientIdAsync(application),
-                Destinations.AccessToken, Destinations.IdentityToken);
```
Comment (kevinchalet) [inline guides/getting-started.md:154]: It's not a valid change, as the `Claim` constructor doesn't take destinations parameters (which is an OpenIddict-specific concept) but `claimValueType` and `issuer` parameters, that have a totally different meaning.

Instead, consider using the one-shot `identity.SetDestinations()` extension: https://documentation.openiddict.com/guides/migration/30-to-40.html#remove-calls-to-addclaims-that-specify-a-list-of-destinations

File: guides/getting-started.md [hunk 2]
```diff
@@ -149,13 +149,13 @@ If you don't want to start from one of the recommended samples, you'll need to:
...
+                Destinations.AccessToken, Destinations.IdentityToken));
 
-            identity.AddClaim(Claims.Name,
+            identity.AddClaim(new Claim(Claims.Name,
```
Comment (kevinchalet) [inline guides/getting-started.md]: Same here.

**Review by kevinchalet — COMMENTED** [review]
Thanks for your PR!

**Comment by TimurZanagar** [comment]
Thank you for the explanation. I have made changes again. What do you think?

**Comment by kevinchalet** [comment]
> Thank you for the explanation. I have made changes again. What do you think?

Looks good, thanks!

**Comment by kevinchalet** [comment]
Merged. Congrats for your first contribution to OpenIddict!

**Pull request merged**